src/components/current_weather_display.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Cloudy from '../images/cloudy.png'
+import Sunny from '../images/sunny.png'
+import Rainy from '../images/rainy.png'
+import Night from '../images/moon.png'
+import NightCloudy from '../images/nightclouds.png'
+import NightRain from '../images/night-rain.png'
+
+class CurrentWeatherDisplay extends React.Component {
+
+toFahrenheit(temp) {
+    return ((temp - 273.15) * 9/5 + 32).toFixed(0) + `\u00B0F`
+}
+
+toCelsius(temp) {
+    return (temp - 273.15).toFixed(0) + `\u00B0C`
+}
+
+render() {
+
+    const { cityName, currentDesc, currentWeather } = this.props 
+  
+    const currTime = (new Date().toLocaleString()).substring(10, 24)
+    let currHour = parseInt(currTime.substring(0,2))
+    let PM = currTime.substring(9,11)
+
+    const display = currentWeather && currentDesc ? (
+            <>
+          <div className="city-name">{cityName}</div>
+            <div className="current-weather">
+                <div className="current-temp">{this.props.Fahrenheit ? this.toFahrenheit(currentWeather.temp) : this.toCelsius(currentWeather.temp)}</div>
+                {(((currHour >= 7 && PM === "PM") || (currHour < 5 && PM === "AM") || (currHour === 12 && PM === "AM"))) ? (
+                        <img className="forecast-img" alt="" src={currentDesc["0"].main === 'Clear' ? Night : currentDesc["0"].main === 'Rain' ? NightRain : NightCloudy}></img> ) : (
+                        <img className="forecast-img" alt="" src={currentDesc["0"].main === 'Clear' ? Sunny : currentDesc["0"].main === 'Rain' ? Rainy : Cloudy}></img> 
+                        )
+                }
+                <div className="current-desc capitalize">{currentDesc["0"].description}</div>
+                <br/>
+                <div>
+                <span className="current-detail">Feels like: {this.props.Fahrenheit ? this.toFahrenheit(currentWeather.feels_like) : this.toCelsius(currentWeather.feels_like)}</span>
+                <span className="current-detail">Humidity: {currentWeather.humidity}%</span>
+                <span className="current-detail">High Today: {this.props.Fahrenheit ? this.toFahrenheit(currentWeather.temp_max) : this.toCelsius(currentWeather.temp_max)}</span>
+                <span className="current-detail">Low Today: {this.props.Fahrenheit ? this.toFahrenheit(currentWeather.temp_min) : this.toCelsius(currentWeather.temp_min)}</span>
+                </div>
+                <span className="last-updated">last updated {currTime}</span>
+            </div>
+            </>
+        ) : (<> </>)
```

Comment:
Is there another way you could render an empty component or nil here?

Reply:
I could simply write "null"

<img width="387" alt="Screen Shot 2020-11-11 at 10 08 39 PM" src="https://user-images.githubusercontent.com/19714255/98890669-7e5ba280-246a-11eb-8734-7856277bbc7e.png">